### ISSUE TITLE
[BUG] Failed to use httpfs extension with DuckDB

### ISSUE THREAD

**Issue by cnwangjie** [DataflareApp/Dataflare#25]
### Describe the bug

Got the following message on executing `LOAD httpfs;`

```
DuckDb("IO Error: Extension "/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension" could not be loaded: dlopen(/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension, 0x0006): tried: '/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension' (code signature in <5CE6D284-D802-3C3D-B84C-F1C9A34B00FE> '/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension' not valid for use in process: mapping process and mapped file (non-platform) have different Team IDs), '/System/Volumes/Preboot/Cryptexes/OS/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension' (no such file), '/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension' (code signature in <5CE6D284-D802-3C3D-B84C-F1C9A34B00FE> '/Users/wangjie/.duckdb/extensions/v0.9.1/osx_arm64/httpfs.duckdb_extension' not valid for use in process: mapping process and mapped file (non-platform) have different Team IDs)")
```

### Platform and Database

```text
macOS 14.0 / DuckDB 0.9.1
```

**Comment by wyhaya** [comment]
@cnwangjie  Thanks for your feedback!

The current DuckDB in Dataflare is very unstable, and there are some other issues that I'm having a hard time fixing, so to prevent causing some unintended errors, I've decided to remove DuckDB from Dataflare starting with the next release.

If possible I'll add it again after fixing these issues.

**Comment by wyhaya** [comment]
@cnwangjie We just released `v1.5.0`, which fixes most errors. If you're still interested in Dataflare, feel free to give it another try!
